### PR TITLE
WDPD-119 add header fields

### PR DESCRIPTION
**Pull request by ps-cgrach**
### This PR

Add header fields with shop system, shop version, plugin name and plugin version

### JIRA Ticket

https://jira.parkside.at/browse/WDPD-119

### CODE REVIEW

File: Core/Helper.php [hunk 1]
```diff
@@ -9,12 +9,14 @@
 
 namespace Wirecard\Oxid\Core;
 
-use OxidEsales\Eshop\Core\Registry;
-use OxidEsales\Eshop\Core\Model\ListModel;
-use OxidEsales\Eshop\Application\Model\Payment;
-
-use Exception;
 use DateTime;
+use Exception;
```
Comment (ps-amehic) [inline Core/Helper.php]: Add a blank line after Exception (use grouping)

File: Core/Helper.php [hunk 2]
```diff
@@ -290,4 +299,26 @@ public static function isThisModule($sModuleId)
...
     }
+
+    /**
+     * @return array
+     *
+     * @since 1.0.0
+     */
+    public static function getShopInfoFields()
+    {
+        $shopId = Registry::getConfig()->getShopId();
```
Comment (ps-amehic) [inline Core/Helper.php]: Rename variable names according to Oxid standards ($shopId -> $sShopId ...)

File: Extend/Model/PaymentGateway.php [hunk 3]
```diff
@@ -9,40 +9,33 @@
 
 namespace Wirecard\Oxid\Extend\Model;
 
-use OxidEsales\Eshop\Application\Model\Shop;
```
Comment (ps-amehic) [inline Extend/Model/PaymentGateway.php]: Same as above (group uses)

File: Core/Helper.php (hunk 1 above)
Comment (ps-cgrach) [reply Core/Helper.php]: :+1:

File: Core/Helper.php (hunk 2 above)
Comment (ps-cgrach) [reply Core/Helper.php]: 👍

File: Extend/Model/PaymentGateway.php (hunk 3 above)
Comment (ps-cgrach) [reply Extend/Model/PaymentGateway.php]: 👍

File: Core/Helper.php [hunk 4]
```diff
@@ -290,4 +299,26 @@ public static function isThisModule($sModuleId)
...
+        $shop = oxNew(Shop::class);
+        $shop->load($shopId);
+
+        $oModule = oxNew(Module::class);
+        $oModule->load(Helper::MODULE_ID);
+
+        return [self::SHOP_SYSTEM_KEY => "OXID",
```
Comment (ps-ohanel) [inline Core/Helper.php]: Add to new line and maybe put the OXID string into constant, as we have "wdoxidee" module ID?

Comment (ps-cgrach) [reply Core/Helper.php]: 👍

**Pull request merged**